.github/workflows/ci-golang-lint.yml

```diff
@@ -19,15 +19,23 @@ jobs:
     name: lint
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
+      - name: Get Go Version
+        run: |
+          #!/bin/bash
+          GOVERSION=$({ [ -f .go-version ] && cat .go-version; })
+          echo "GOVERSION=$GOVERSION" >> $GITHUB_ENV
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GOVERSION }}
       - name: golang-lint
         env:
           # prevent OOM
           GOGC: 10
-        uses: golangci/golangci-lint-action@v2
+        uses: golangci/golangci-lint-action@v3.2.0
```

Comment:
should we just do @latest here?

Reply:
if it does break, we would know something changed in the action side. Times when this would break is when the underlying schema would change things the way this action is invoked or some new fields coming in. 

Irrespective it's good to do latest here as well IMO

Reply:
Makes sense!   I pinned it to @ latest

Reply:
Interesting, @ latest is not possible ([see the failed run here](https://github.com/dgraph-io/dgraph/actions/runs/3396771636)).  But @ v3 works and will pull the latest minor versions

Reply:
they do have a `latest` tag https://github.com/golangci/golangci-lint-action/releases <- here

Reply:
you can probably just do `golangci/golangci-lint-action` without a version, this is still a guess